src/Homely.AspNetCore.Mvc.Helpers/Extensions/MvcOptionsExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Homely.AspNetCore.Mvc.Helpers.Filters;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Authorization;
+
+namespace Homely.AspNetCore.Mvc.Helpers.Extensions

```

Comment:
Note: need to test this out to see how it works with our current JSON exception handling.